ale_linters/go/gobuild.vim

```diff
@@ -39,15 +40,14 @@ function! ale_linters#go#gobuild#GetMatches(lines) abort
 endfunction
 
 function! ale_linters#go#gobuild#Handler(buffer, lines) abort
+    let l:dir = expand('#' . a:buffer . ':p:h')
     let l:output = []
 
     for l:match in ale_linters#go#gobuild#GetMatches(a:lines)
-        " Omit errors from imported go packages
-        if !ale#path#IsBufferPath(a:buffer, l:match[1])
-            continue
-        endif
 
+        echom l:match[1]
```

Comment:
You left an echo here.

Reply:
I just pushed a commit so TravisCI will complain about stray echo lines in future.